config/analytics.yml

```diff
@@ -7,6 +7,6 @@ analytics:
     app_name: <%= ENV['GOOGLE_OAUTH_APP_NAME'] %>
     app_version: <%= ENV['GOOGLE_OAUTH_APP_VERSION'] %>
     privkey_value: <%= ENV['GOOGLE_OAUTH_PRIVATE_KEY_VALUE'] %>
-    # OR privkey_path: <%= ENV['GOOGLE_OAUTH_PRIVATE_KEY_PATH'] %>
+    privkey_path: <%= ENV['GOOGLE_OAUTH_PRIVATE_KEY_PATH'] %>
```

Comment:
since we are leaving the possibility of setting `self.google_oauth_private_key_value` in `#reload_analytics`, we need the app to know what `config.privkey_path` is.